git-imerge

```diff
@@ -3629,9 +3629,22 @@ def cmd_diagram(parser, options):
 
 
 def cmd_reparent(parser, options):
+    if len(options.parents) == 0:
+        parser.error('Parent is not specified, please specify at least one parent.')
```

Comment:
Why do you add this new restriction? It is perfectly sensible to set a commit to have zero parents; in that case, the commit becomes an orphan commit (i.e., like the first commit made in a fresh repository).